venues/ICLR.cc/2019/Conference/python/post-submissions-stage.py

```diff
@@ -34,7 +34,7 @@
 
     client.post_invitation(iclr19.blind_submission_inv)
 
-    original_notes = openreview.tools.iterget_notes(client, invitation=iclr19.submission_inv.id)
```

Comment:
Why did you change this line?

Reply:
I'm guessing this was unintentional, since it breaks the indentation